apps/src/code-studio/components/stageExtras/MazeThumbnail.jsx

```diff
@@ -5,7 +5,7 @@ import {getSubtypeForSkin} from '@cdo/apps/maze/mazeUtils';
 import MazeMap from '@cdo/apps/maze/mazeMap';
 import drawMap from '@cdo/apps/maze/drawMap';
 
-const assetUrl = path => '/blockly/' + path;
+const assetUrl = path => 'https://studio.code.org/blockly/' + path;
```

Comment:
Found in https://code-dot-org.github.io/cdo-styleguide/static/preview.9c1be315b2e8e071c8a4.bundle.js:

![screen shot 2017-07-07 at 11 02 05 am](https://user-images.githubusercontent.com/413693/27970481-e4d6d95c-6303-11e7-9330-99ccb4212ca8.png)

Does this mean the deployed Storybook is in `"production"` mode?  Thanks!